pkg/connector/slack/v0/config/tasks.json

```diff
@@ -0,0 +1,235 @@
+{
+    "TASK_READ_MESSAGE": {
+      "instillShortDescription": "Get the latest message since specific date",
+      "input": {
+        "description": "Please input the channel name and the date that we want to start to read",
+        "instillUIOrder": 0,
+        "properties": {
+          "channel_name": {
+            "description": "A channel name display in Slack",
+            "instillAcceptFormats": [
+              "string"
+            ],
+            "instillUIMultiline": true,
+            "instillUIOrder": 0,
+            "instillUpstreamTypes": [
+              "value"
```

Comment:
Sorry I put my comment in wrong place, please allow using reference and template here.